### DISCUSSION TITLE
Admin Coverage H2H Day 1-4 Campaign Total Vaccinated is not correct

### DISCUSSION THREAD

**Discussion by naveeddil** [xlg8/APMIS-Project#669] [General]
Dear Team,

I tried to check the formula from form manager and it is fine as 
TotalChildrenVaccinated = (TotalChildrenVaccinated_day1_3GT + TotalChildrenVaccinated_day4GT)
But the column show an incorrect sum.

The blue highlighted are equal to 1308 although the green highlighted shows a discrepancy.

![image](https://github.com/AFG-Polio-Data/APMIS-Project/assets/54671296/18162afa-e1f5-4d40-84f4-9bfdfa76bd33)

@bnygren @RaziaMahmodi

**Comment by bnygren** [comment]
Ok,. thank you for noting this. I will follow up.

**Comment by bnygren** [comment]
@RaziaMahmodi is checking this.
